src/sections/Meshery/Meshery-integrations/Individual-Integrations/index.js

```diff
@@ -4,16 +4,16 @@ import Button from "../../../../reusecore/Button";
 import { MDXRenderer } from "gatsby-plugin-mdx";
 import CTA_Book from "./cta-book";
 import RelatedIntegration from "./relatedIntegration";
-import Image from "../../../../components/image";
 import HowIntegrationWorks from "./howItWork";
 import { IntegrationPageWrapper } from "./individual-integrations.style";
 
+
 const IndividualIntegrations = ({ data }) => {
   const { frontmatter, body } = data.mdx;
   return (
     <IntegrationPageWrapper>
       <section className="herosection">
-        <Image {...frontmatter.integrationIcon} imgStyle={{ objectFit: "contain" }} alt={frontmatter.title}/>
+        <img src={frontmatter.integrationIcon.publicURL} alt={frontmatter.title} height={90}/>
```

Comment:
Why switch from gatsby-image?

Reply:
It was not showing the image in the hero section. That's why I replaced that with a standard image tag.

Reply:
Then, it might be a CSS issue. Did you check what was the issue?

Reply:
I checked that, and it wasn't a CSS issue cause the only thing CSS changed was the position of that image. So I doubt it was related to the CSS issue.

Reply:
![image](https://user-images.githubusercontent.com/34382211/184064423-feebf468-fe7a-4992-a9ee-c8c6696125e8.png)

Reply:
gatsby-image doesn't render the img tag directly. Ut wraps around a div. So, check that while inspecting the code in the dev tools

Reply:
I have fixed that issue.

Reply:
Nice! 🚀